app/webpacker/styles/lead_providers.scss

```diff
@@ -0,0 +1,537 @@
+.index-group-lead-provider {
```

Comment:
I'm surprised we need this much styling. Are we using the gov design system fully? Is this all handrolled?

Reply:
I'll clean it up

Reply:
Deleted anything not used